src/interface/src/app/plan/project-areas/project-areas.component.html

```diff
@@ -0,0 +1,31 @@
+<div class="flex-row">
+  <h3 class="title">Project Areas</h3>
+  <mat-icon class="material-symbols-outlined" color="primary">info</mat-icon>
+</div>
+<table>
+  <thead>
```

Comment:
I went for an old school table for this and not the angular material one because it felt like an overkill. I rather have simpler markup if there isn't any special interaction

Reply:
Yeah I'm all for that, we don't need anything fancy for this